modules/message-lane/src/benchmarking.rs

```diff
@@ -118,7 +118,81 @@ benchmarks_instance! {
 		confirm_message_delivery::<T, I>(T::MaxMessagesToPruneAtOnce::get());
 
 		let (payload, fee) = T::prepare_outbound_message(MessageParams {
-			size_factor: WORST_MESSAGE_SIZE_FACTOR,
+			size: 0,
+			sender_account: sender.clone(),
+		});
+	}: send_message(RawOrigin::Signed(sender), lane_id, payload, fee)
+	verify {
+		assert_eq!(
+			crate::Module::<T, I>::outbound_latest_generated_nonce(bench_lane_id()),
+			T::MaxMessagesToPruneAtOnce::get() + 1,
+		);
+	}
+
+	// Benchmark `send_message` extrinsic with the worst possible conditions:
+	// * outbound lane already has state, so it needs to be read and decoded;
+	// * relayers fund account does not exists (in practice it needs to exist in production environment);
+	// * maximal number of messages is being pruned during the call;
+	// * message size is 1KB.
+	//
+	// With single KB of message size, the weight of the call is increased (roughly) by
+	// `(send_16_kb_message_worst_case - send_1_kb_message_worst_case) / 16`.
```

Comment:
```suggestion
	// `(send_16_kb_message_worst_case - send_1_kb_message_worst_case) / 15`.
```

Shouldn't this be 15?